pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -10012,33 +10024,38 @@ spec:
 	tamperedPipeline.Annotations["random"] = "attack"
 
 	testCases := []struct {
-		name        string
-		pipelinerun []*v1beta1.PipelineRun
-		pipeline    []*v1beta1.Pipeline
-		task        []*v1beta1.Task
+		name          string
+		pipelinerun   []*v1beta1.PipelineRun
+		pipeline      []*v1beta1.Pipeline
+		task          []*v1beta1.Task
+		wantCondition *apis.Condition
 	}{
 		{
-			name:        "unsigned pipeline fails verification",
-			pipelinerun: []*v1beta1.PipelineRun{prs},
-			pipeline:    []*v1beta1.Pipeline{ps},
+			name:          "unsigned pipeline fails verification",
+			pipelinerun:   []*v1beta1.PipelineRun{prs},
```

Comment:
it might be helpful to name this variable `unsignedPipelineRun` instead of `prs`, and likewise for other test inputs